service/controller/app/v1/resource_set.go

```diff
@@ -155,11 +155,13 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 			return nil, microerror.Mask(err)
 		}
 
-		// TODO: Remove if statement after in-cluster flag is implemented.
 		var k8sClient kubernetes.Interface
 		var g8sClient versioned.Interface
 
-		if cr.Spec.KubeConfig.Secret.Name != "" {
+		if key.InCluster(cr) {
```

Comment:
Can `InCluster` related changes land in a separate PR?

Reply:
No, Since it's already in master after your initial review, it's gone now.

Reply:
Sorry I don't follow. You mean it will disappear after merging master branch?

Reply:
My bad; I just rebase it with master and it's gone :)